tsh.cc

```diff
@@ -36,16 +36,16 @@ static char prompt[] = "tsh> ";
 int verbose = 0;
 static char childprocs = 0;
 
-struct child 
+struct child
```

Comment:
This is faster and more idiomatic to a systems program. However, it may have caused changes in the behavior of the program I didn't account for. Lets go over this.